src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php

```diff
@@ -327,7 +327,7 @@ public function testReduce()
     {
         $crawler = $this->createTestCrawler()->filterXPath('//ul[1]/li');
         $nodes = $crawler->reduce(function ($node, $i) {
-            return $i == 1 ? false : true;
+            return $i != 1;
```

Comment:
as it's a test, it wouldn't hurt that much (and since `$i` is the loop variable, it will always be an integer)